src/client/node/nodeKwil.ts

```diff
@@ -65,34 +82,64 @@ export class NodeKwil extends Kwil<EnvironmentType.NODE> {
         actionBody.inputs[0] instanceof ActionInput
           ? (actionBody.inputs as ActionInput[])
           : new ActionInput().putFromObjects(actionBody.inputs as Entries[]);
-      msg = msg.concat(inputs);
+      msg1 = msg1.concat(inputs);
     }
 
     if (kwilSigner) {
-      msg = msg
+      msg1 = msg1
         .signer(kwilSigner.signer, kwilSigner.signatureType)
         .publicKey(kwilSigner.identifier);
     }
 
-    const message = await msg.buildMsg();
+    // message without signature
+    let message = await msg1.buildMsg();
+    // @ts-ignore
+    delete message.data.sender;
```

Comment:
Remove this ts-ingore and the delete. You are going to need to refactor the builder so that you can make call requests without the sender,